packages/react-magma-dom/src/components/NativeSelect/NativeSelect.test.js

```diff
@@ -65,6 +65,20 @@ describe('NativeSelect', () => {
     );
   });
 
+  it('should update the selected option', () => {
+    const { getByTestId, getAllByTestId } = render(
+      <NativeSelect testId={testId}>
+        <option data-testid={`${testId}-option-1`}>1</option>
+        <option data-testid={`${testId}-option-2`}>2</option>
+        <option data-testid={`${testId}-option-3`}>3</option>
+      </NativeSelect>
+    );
+    fireEvent.change(getByTestId(testId), { target: { value: 2 } });
+    expect(getByTestId(`${testId}-option-1`).selected).toBeFalsy();
+    expect(getByTestId(`${testId}-option-2`).selected).toBeTruthy();
+    expect(getByTestId(`${testId}-option-3`).selected).toBeFalsy();
```

Comment:
Do options have `selected`? Why check for falsy/truthy instead of true/false?

Reply:
Changed testing method.